rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -443,15 +452,12 @@ impl RoadBuilder {
             let middle_separator = match [self.forward_inside(), self.backward_inside()] {
                 [Some(forward), Some(backward)] => {
                     lanes_to_separator([forward, backward], &self, tags, locale, warnings)
-                }
+                },
                 [Some(lane), None] | [None, Some(lane)] => {
                     lane_to_inner_edge_separator(lane.mirror()).map(Lane::mirror)
-                }
-                _ => todo!(),
+                },
+                [None, None] => return Err(RoadError::Msg(RoadMsg::Internal("no lanes"))),
             };
-            let forward_edge = dbg!(forward_edge);
```

Comment:
Hmm, you removed it here too, I wonder if the benchmark presubmit check didn't pick this up?